### ISSUE TITLE
Audit colors

### ISSUE THREAD

**Issue by afercia** [Yoast/License-Manager#95]
As part of the general colors review to improve accessibility, some colors that are declared inline (red, limegreen, white) should be updated with new colors.
There are a very few occurrences of inline styles (I can count 4 of them) so probably there are two options:
- easier: just update the colors with accessible ones
- better: introduce the planned colors palette

Feedback welcome :)
